app/controllers/theblog/content_nodes_controller.rb

```diff
@@ -7,6 +7,10 @@ def show
         by_parent(params[:category]).
         find_by(slug: params[:slug])
 
+      if @node.is_a? Theblog::Category
+        @posts = @node.child_nodes.published.order("created_at DESC").page params[:page]
```

Comment:
I'd add it like 
`@node.child_nodes.published.order("created_at DESC").page(params[:page])`